returnn/tf/util/data.py

```diff
@@ -295,6 +295,23 @@ def get_uniq_collection(cls, tags, is_equal_opts=None):
         res.append(tag)
     return res
 
+  def get_unbroadcast_dim(self):
```

Comment:
(Meta: This is somewhat annoying to review on GitHub, because you don't see which class this belongs too... (Here `DimensionTag`.) I currently expand it until I see the class. Not sure what to do about it. Maybe tell GitHub as a feature request to show the class.)

The function name is not good. Why "unbroadcast"? You are simply returning the dimension here. Maybe `get_dimension`. Or maybe `get_dim_value`. Or so.

Reply:
(About GitHub: We can [file a suggestion here](https://support.github.com/contact/feedback) if it is annoying. I personally have never reviewed anything yet though)

About the name: Yes. Let's go for `get_dim_value` then, I think `get_dimension` is a little bit confusing as there is also the attribute `dimension`.